wlm/src/main/java/ai/djl/serving/wlm/LmiUtils.java

```diff
@@ -54,6 +52,21 @@ public final class LmiUtils {
 
     private LmiUtils() {}
 
+    static void exec(List<String> cmd) throws IOException, InterruptedException {
```

Comment:
We can do this as a follow up, but might be nice to put this into a generic ProcessUtils class or something similar. We have a few more places where we do this (like cluster request handler)

Reply:
We can make this in `ProcessUtils` class, but need to make error checking more generic, this class currently has logic specific for s3cmd:

```
logOutput.startsWith("ERROR ")
```

We can pass a regex if caller need extra error checking:
```
(regex != null && logOutput.matches(regex))
```